SPECS/mote.spec

```diff
@@ -1,15 +1,13 @@
-%{!?python_sitelib: %global python_sitelib %(%{__python} -c "from
```

Comment:
You were redefining it here, albeit now that I look at it, it should be only added when not yet defined.

Reply:
Got it, Thank you very much Dan for the PR. :D